DANS-KNAW/easy-manage-deposit
# EASY-2833 Deposit.properties as a Service

Fixes EASY-2833

# Description of changes
* Adds a one-table database to index the `deposit.properties` text + some properties as separate columns, for faster searching and reporting of totals.
* Adds a simple HTTP-service that lets you load properties into the database by UUID. This is to be used by `easy-sword2` and `easy-ingest-flow` later, to keep the database up-to-date.
* Retains the old reporting code for now, to allow for regression testing.
* The `clean` command still crawls the disk, but now also updates the database, after making changes. 
* Refactors the Configuration class to look more like the other modules.

# How to test
* Deploy on the server using the modified `easy-dtap` (see Related PRs)
* Copy some deposits to the various directories of the deposit area, if not already filled.
* `easy-manage-deposit load-properties` (fills the database)
* `easy-manage-deposit report summary` (`full`, `error`, etc, try the various options)

# Related PRs
* https://github.com/DANS-KNAW/easy-dtap/pull/504

# Notify
* @DANS-KNAW/dataversedans
* @DANS-KNAW/easy
